wwrmi/api.go

```diff
@@ -595,26 +592,24 @@ func (client *Client) Read(p []byte) (copied int, err error) {
 	debugging.DEBUG_OUT("RMI Read() (logs) EOF\n")
 	if copied > 1 {
 		// we ran out of buffers, so say EOF
-		client.sentBytes += uint32(copied-1)
-		copy(p[copied-2:],[]byte("]")) // replace last ',' with ']'
+		client.sentBytes += uint32(copied - 1)
+		copy(p[copied-2:], []byte("]")) // replace last ',' with ']'
 	} else {
 		// it was en empty read. Not what we want (this should not happen)
 		// but let's make it valid JSON
 		if remain > 1 {
-			copy(p[1:],[]byte("]")) // replace last ',' with ']'
+			copy(p[1:], []byte("]")) // replace last ',' with ']'
 			err = errors.New("no data")
 			client.readOngoing = false
 			return
 		}
 	}
-//	debugging.DEBUG_OUT("RMI - READ EOF >>%s<<\n",string(p))
+	//	debugging.DEBUG_OUT("RMI - READ EOF >>%s<<\n",string(p))
```

Comment:
Can you please remove the space after // ?

Reply:
It is not part of my change in code. I probably will stay away from cleaning every comment in code base for now. 
`go fmt` updated the tab/spaces to make it readable, but I am not explicitly updating code comments to make them look neat

Reply:
Can you please update it anyway as it doesn't align with other comments, or you can remove it completely.